### ISSUE TITLE
filter_liveSearch

### ISSUE THREAD

**Issue by mrlioncub** [Mottie/tablesorter#285]
How to search for this option filter_liveSearch is false?

``` html
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN"><HTML><HEAD>
<meta http-equiv="Content-Type" content="text/html; charset=cp1251">
<TITLE>А <-> Выписные заметки</TITLE></HEAD><BODY>
<link href="http://test.domain.loc/tablesorter2.9.1/css/theme.green.css" rel="stylesheet">
<style type="text/css">table.tablesorter tbody tr.selectlines td {background-color:#99FF99</style>
<script type="text/javascript" src="http://test.domain.loc/jquery/jquery-1.9.1.min.js"></script>
<script type="text/javascript" src="http://test.domain.loc/tablesorter2.9.1/js/jquery.tablesorter.min.js"></script>
<script type="text/javascript" src="http://test.domain.loc/tablesorter2.9.1/addons/pager/jquery.tablesorter.pager.min.js"></script>
<script type="text/javascript" src="http://test.domain.loc/tablesorter2.9.1/js/jquery.tablesorter.widgets.min.js"></script>
<script type="text/javascript">
    $(document).ready(function() {
    $('#table')
    .tablesorter({showProcessing: true, theme : 'green', sortList:[], widthFixed: true, widgets: ['zebra','filter'], widgetOptions : {filter_liveSearch: false}})
    .tablesorterPager({container: $("#pager")});
    });
</script><TABLE id="table" class="tablesorter-green"><THEAD><TR><TD><B>Файл</B></TD><TD><B>Отделение</B></TD><TD><B>Год</B></TD></TR></THEAD> <TBODY><TR><TD><a href="1 отделение\Аааааа В И 13.12.2011.doc">Аааааа В И 13.12.2011.doc</TD><TD>1 отделение</TD><TD>+2011</TD></TR>
<TR><TD><a href="н отделение\Абааааа П.М..doc">Абааааа П.М..doc</TD><TD>н отделение</TD><TD>+2010</TD></TR>
<TR><TD><a href="Н отделение\Абаба.doc">Абаба.doc</TD><TD>Н отделение</TD><TD>+2009</TD></TR>
<TR><TD><a href="Г отделение\Абае 16_8_2010.doc">Абае 16_8_2010.doc</TD><TD>Г отделение</TD><TD>+2010</TD></TR>
<TR><TD><a href="У отделение\Абаи ИП_2012_xi_30.doc">Абаи ИП_2012_xi_30.doc</TD><TD>У отделение</TD><TD>+2012</TD></TR>
<TR><TD><a href="Отделение Р\Абао  Н_2012_ix_14.doc">Абао  Н_2012_ix_14.doc</TD><TD>Отделение Р</TD><TD>+2012</TD></TR>
</TBODY></TABLE><div id="pager" class="pager" style="position: absolute; "><form>
<img src="http://test.domain.loc/tablesorter2.9.1/addons/pager/icons/first.png" class="first"><img src="http://test.domain.loc/tablesorter2.9.1/addons/pager/icons/prev.png" class="prev">
<input type="text" class="pagedisplay">
<img src="http://test.domain.loc/tablesorter2.9.1/addons/pager/icons/next.png" class="next"><img src="http://test.domain.loc/tablesorter2.9.1/addons/pager/icons/last.png" class="last">
<select class="pagesize"><option selected="selected" value="10">10</option><option value="50">50</option><option value="100">100</option></select>
</form></div></BODY></HTML>
```

this search doesn't work...
Please show an example.

**Comment by ashum81** [comment]
filter_liveSearch: false is not workng in last chrome
The column search starts after typing.

**Comment by Mottie** [comment]
Hi @ashum81!

This was fixed in the last release - see #1134. Please make sure you're using the latest version.

**Comment by ashum81** [comment]
Thank you! I worked with version 2.25.2. After update filter works fine.


>Пятница, 29 января 2016, 5:24 -08:00 от Rob G <notifications@github.com>:
>
>Hi  @ashum81 !
>This was fixed in the last release - see  #1134 . Please make sure you're using the latest version.
>—
>Reply to this email directly or  view it on GitHub .



